tests/namespaces.rs

```diff
@@ -28,6 +28,11 @@ fn namespace() {
             e
         ),
     }
+    assert_eq!(
+        r.prefixes().collect::<Vec<_>>(),
+        vec![(PrefixDeclaration::Named(b"myns"), Namespace(b"www1"))]
+    );
+
```

Comment:
I think, you should add those checks after each `read_resolved_event()` in all places where you've add that check

Reply:
Done!